.travis.yml

```diff
@@ -1,11 +1,19 @@
 language: node_js
 node_js:
-  - "8"
-  - "9"
+- '8'
+- '9'
+- '10'
 script:
-  - yarn test
-
+- yarn test
 notifications:
   email:
     on_success: never
     on_failure: always
+deploy:
+  provider: npm
+  email: $NPM_EMAL
+  api_key: $NPM_KEY
```

Comment:
These are configured with my credentials for now. Somebody should probably update these with the `lifeomicdev` credentials.